### PR TITLE
Add table of supported eBPF instructions

### PR DESCRIPTION
**Pull request by dthaler**
Signed-off-by: Dave Thaler <dthaler@microsoft.com>

## Description

Add table of supported eBPF instructions.
This is part of #1451

## Documentation

This is a doc change.

### CODE REVIEW

**Comment by Alan-Jowett** [comment]
It would be cool if we could generate the list as part of the [gh-pages build step](https://github.com/microsoft/ebpf-for-windows/blob/main/.github/workflows/update-docs.yml) so we don't have to worry about updating it.

**Comment by dthaler** [comment]
> It would be cool if we could generate the list as part of the [gh-pages build step](https://github.com/microsoft/ebpf-for-windows/blob/main/.github/workflows/update-docs.yml) so we don't have to worry about updating it.

That's possibly doable but hard, so left for a future PR.

**Review by saxena-anurag — APPROVED** [review]

File: docs/isa-support.rst [hunk 1]
```diff
@@ -0,0 +1,222 @@
+eBPF Instruction Set
+====================
+
+The following table shows which
+[eBPF instructions](https://github.com/dthaler/ebpf-docs/blob/update/isa/kernel.org/instruction-set.rst)
+are currently supported by PREVAIL, uBPF, and bpf2c, and which bpf_conformance test covers it.
+
+======  ===  ====  ===================================================  =======  ====  =====  ======================
+opcode  src  imm   description                                          PREVAIL  uBPF  bpf2c  conformance
```
Comment (shankarseal) [inline docs/isa-support.rst:9]: `conformance test` ?

File: docs/isa-support.rst [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+0x05    0x0  0x00  goto +offset                                            Y      Y      Y    exit-not-last
+0x07    0x0  any   dst += imm                                              Y      Y      Y    add64
+0x0c    any  0x00  dst = (uint32_t)(dst + src)                             Y      Y      Y    add
+0x0f    any  0x00  dst += src                                              Y      Y      Y    alu64-arith
+0x14    0x0  any   dst = (uint32_t)(dst - imm)                             Y      Y      Y    alu-arith
+0x15    0x0  any   if dst == imm goto +offset                              Y      Y      Y    jeq-imm
+0x16    0x0  any   if (uint32_t)dst == imm goto +offset                    Y      Y      no   jeq32-imm
+0x17    0x0  any   dst -= imm                                              Y      Y      Y    alu64-arith
+0x18    0x0  any   dst = imm64                                             Y      Y      Y    lddw
+0x18    0x1  any   dst = map_by_fd(imm)                                    no     no     no   ???
```
Comment (shankarseal) [inline docs/isa-support.rst:22]: `N/A` instead of `???`

**Review by shankarseal — APPROVED** [review]

**Pull request merged**